packages/utils/scaffold.ts

```diff
@@ -11,6 +11,29 @@ import astTransform from "./recursive-parser";
 import runPrettier from "./run-prettier";
 import { Node } from "./types/NodePath";
 
+
+
+function mergeHandler(config: Config, transformations: string[]): [Config, string[]]{
```

Comment:
I'm not a fan of this function as it isn't descriptive enough and it passes the transformation object in an indrection we don't want the AST to use. The transformation and it's data should have a 1:1 mapping

Reply:
Remove this function